### DISCUSSION TITLE
Upload test screenshots with unknown name

### DISCUSSION THREAD

**Discussion by cecilian** [community/community#25169] [Actions]
<p>Hello,</p>
<p>I’m setting up a CI workflow and I would like to save screenshots of failing builds using the <code>upload-artifacts</code> action.</p>
<p>However the name of the screenshot file is not known beforehands (for instance, it will contain a timestamp) so I cannot hardcode it in the .yml</p>
<p>Is there a way around this problem?</p>
<p>I have looked at this question  <a href="https://github.community/t/github-actions-using-non-fixed-names-in-upload-artifact/17353">https://github.community/t5/GitHub-Actions/GitHub-Actions-Using-non-fixed-names-in-upload-artifact/td-p/43161</a> but unfortunately the solution is not applicable for my case.</p>
<p>Thank you</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/cecilian">@cecilian</a> ,</p>
<p>‘Path’ is required for action ‘actions/upload-artifact’ and not accpet pattern ‘*’. Hence, after you capature the screenshots, you can put them in a subfolder under root, specify the folder as the path value.</p>
<pre><code>- name: Upload screenshots
        uses: actions/upload-artifact@v1
        with:
          name: screenshots
          path: ./scr #Put your screenshots in subfolder ./src before upload
</code></pre>
<p>Hope it helps!</p>
